edx_solutions_api_integration/users/serializers.py

```diff
@@ -206,3 +207,20 @@ def get_course(self, enrollment):
             ), None
         )
         return course_overview
+
+
+class ExtendedOrganizationSerializer(BasicOrganizationSerializer):
+    """ Serializer for Organization with mobile apps and themes data """
+    mobile_apps = MobileAppSerializer(many=True)
+    themes = serializers.SerializerMethodField()
+
+    class Meta(BasicOrganizationSerializer.Meta):
+        fields = BasicOrganizationSerializer.Meta.fields + ('themes', 'mobile_apps')
+
+    def get_themes(self, organization):
+        themes = [theme for theme in organization.theme.all() if theme.active]
+        serializer = ThemeSerializer(themes, many=True)
```

Comment:
Since one organization is going to have only one active theme `many` does not make sense here. and name of the method should be `get_theme`.